templates/main.html

```diff
@@ -1,22 +1,33 @@
 <!DOCTYPE html>
-<html lang="en">
+<html>
```

Comment:
lang="mul".

If not, it should still be `lang=en`, because it is only with JavaScript enabled that it is multilingual, and the JavaScript should update the lang attribute as required.